fields.fpp

```diff
@@ -1363,6 +1769,7 @@ contains
 
    end subroutine get_radial_correction
 
+   !> compute d<chi>/dy in (ky,kx,z,tube) space
```

Comment:
Could use `\(...\)` to write LaTeX in FORD docstrings. This makes the documentation prettier, though at the cost of making the in-code comment harder to read, depending on the LaTeX!